devlab/provision/devstack/local_juno.conf

```diff
@@ -0,0 +1,54 @@
+[[local|localrc]]
+ADMIN_PASSWORD=admin
+DATABASE_PASSWORD=secret
+RABBIT_PASSWORD=$DATABASE_PASSWORD
+SERVICE_PASSWORD=$ADMIN_PASSWORD
+SERVICE_TOKEN=a682f596-76f3-11e3-b3b2-e716f9080d50
+
+HOST_IP=192.168.1.8
+
+#Neutron
+FLOATING_RANGE=192.168.1.0/24
+FLAT_INTERFACE=eth1
+PUBLIC_NETWORK_GATEWAY=$HOST_IP
+Q_FLOATING_ALLOCATION_POOL='start=192.168.1.100,end=192.168.1.254'
+Q_PLUGIN=ml2
+Q_AGENT_EXTRA_AGENT_OPTS=(tunnel_type=gre)
+Q_AGENT_EXTRA_OVS_OPTS=(tenant_network_type=gre)
+Q_SRV_EXTRA_OPTS=(tenant_network_type=gre)
+
+
+
+#Log
+ENABLE_DEBUG_LOG_LEVEL=False
+VERBOSE=True
+DEST=/opt/stack
+LOGDIR=/home/vagrant/logs/services
+SCREEN_LOGDIR=/home/vagrant/logs/screen
+LOGFILE=/home/vagrant/logs/stack.sh.log
+LOG_COLOR=False
+
+
+
+# Pre-requisite
+ENABLED_SERVICES=rabbit,mysql,key
+
+# Horizon (always use the trunk)
+ENABLED_SERVICES+=,horizon
+
+# Nova
+ENABLED_SERVICES+=,n-api,n-crt,n-obj,n-cpu,n-cond,n-sch,n-novnc,n-cauth
+IMAGE_URLS+=",https://launchpad.net/cirros/trunk/0.3.0/+download/cirros-0.3.0-x86_64-disk.img"
+
```

Comment:
Please add
> NOVA_INSTANCES_PATH=/var/lib/nova/instances

Because by default it use `/opt/stack/data/nova/instances` and migration does not works. It root cause for SAIC-321